scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  http_basic_authenticate_with name: "dhh", password: "secret", only: :destroy
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create!(comment_params)
+    redirect_to @post
+  end
+
+  def destroy
+    @post = Post.find(params[:post_id])
+    @post = @post.comments.find(params[:id])
+    @comment.destroy
+    redirect_to (@post)
```

Comment:
it's best not to put a space between a method and the parens with it's arguments. can lead to errors / bugs in some situations